composer.json

```diff
@@ -69,9 +69,7 @@
             "drupal/core": {
                 "Contextual links should not be added inside another link - https://www.drupal.org/node/2898875": "https://www.drupal.org/files/issues/2018-03-16/contextual_links_should-2898875-6.patch",
                 "RSS view with fields give wrong URL from path field - https://www.drupal.org/node/2673980#comment-12180229": "https://www.drupal.org/files/issues/rss_view_with_fields-2673980-28.patch",
-                "DateTimeIso8601::getDateTime() does not specify the storage timezone when constructing DrupalDateTime object - https://www.drupal.org/project/drupal/issues/3002164": "https://www.drupal.org/files/issues/2018-09-25/3002164-7.patch",
                 "Custom blocks are no longer displayed and no longer appear in the custom block library - https://www.drupal.org/project/drupal/issues/2997898#comment-12832813": "https://www.drupal.org/files/issues/2018-10-27/2997898-21.patch",
-                "Make the DateTime input format and descriptive text consistent - https://www.drupal.org/project/drupal/issues/2791693": "https://www.drupal.org/files/issues/2018-06-22/2791693-datetime-input-ui-29.patch",
```

Comment:
@vincent-gao I assume this didn't apply... did we check if there was an updated patch or is it no longer required?

Reply:
1. 
```
"Make the DateTime input format and descriptive text consistent - https://www.drupal.org/project/drupal/issues/2791693": "https://www.drupal.org/files/issues/2018-06-22/2791693-datetime-input-ui-29.patch"
```
was committed to drupalcore 8.7.x no need for this patch.